modules.d/99base/rdsosreport.sh

```diff
@@ -27,8 +27,8 @@ done
 cat /proc/self/mountinfo
 cat /proc/mounts
 
-blkid
-blkid -o udev
+command -v blkid > /dev/null 2> /dev/null && blkid
```

Comment:
Why this extra `command -v blkid`? don't we always require `util-linux`?

Reply:
I only pointed out that the `command -v blkid` check is not necessary, but the `command -v udevadm` is required here.